.pfnci/run.sh

```diff
@@ -77,8 +86,10 @@ main() {
   git config --global user.email "you@example.com"
   git config --global user.name "Your Name"
 
+  # Xvfb's default screen is 1280x1024x8, which seems to cause a problem.
+  # https://bugzilla.redhat.com/show_bug.cgi?id=904851
   OMP_NUM_THREADS=1 PYTHONHASHSEED=0 \
-      xpytest "${xpytest_args[@]}" '/chainerrl/tests/**/test_*.py'
+      xvfb-run --server-args="-screen 0 1280x800x24" xpytest "${xpytest_args[@]}" '/chainerrl/tests/**/test_*.py'
```

Comment:
How about improving the readability by
```suggestion
      xvfb-run --server-args="-screen 0 1280x800x24" \
      xpytest "${xpytest_args[@]}" '/chainerrl/tests/**/test_*.py'
```